far/viewer.cpp

```diff
@@ -851,14 +897,17 @@ void Viewer::ShowDump()
 
 void Viewer::ShowHex()
 {
-	const auto HexLeftPos = ((LeftPos > 80 - ObjWidth())? std::max(80 - ObjWidth(), 0) : LeftPos);
-	const string BorderLine{ BoxSymbols[BS_V1], L' '};
+	const auto HexLeftPos = ((LeftPos > 80 - ObjWidth()) ? std::max(80 - ObjWidth(), 0) : LeftPos);
 
 	int tail = 0;
 	bool EndFile = false;
 
 	LastPage = false;
 
+	if (m_PrevXX2 != XX2)
+		m_PrevXX2 = XX2,
+		m_BytesPerLine = std::min(CalculateMaxBytesPerLineByScreenWidth(), m_BytesPerLine);
```

Comment:
Does it mean that when Viewer is opened for the first time, it will display as many bytes per line as possible? Maybe it is better to keep the default 16 bytes per line for the start? Later an option can be added to control the default hex view. Just asking. My apologies if I misunderstood the implementation.

Reply:
+1, let's keep the conventional behaviour by default.